daemon/containerd/image_list.go

```diff
@@ -66,63 +70,32 @@ func (i *ImageService) Images(ctx context.Context, opts types.ImageListOptions)
 		root = make([]*[]digest.Digest, len(imgs))
```

Comment:
The slice size is too small for the `root[n+m] = &chainIDs` (will panic). Perhaps we should just use `append` and reserve capacity instead of setting the length upfront.

```suggestion
		root = make([]*[]digest.Digest, 0, len(imgs))
```

(see comment 3 next)